CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   changed as well. [cyberark/conjur-puppet#13](https://github.com/cyberark/conjur-puppet/issues/13).
 - `conjur::secret` Optional parameters now use a Hash instead of positional parameters.
   [cyberark/conjur-puppet#184](https://github.com/cyberark/conjur-puppet/issues/184).
+- User experience for situations where Conjur identity cannot be resolved has been
+  impoved with better errors/warnings.
```

Comment:
```suggestion
  improved with better errors/warnings.
```
issue link?

Reply:
I'll fix the spelling. There's no issue for the fixup as it was just a result from the extra tests. Want me to put down the current issue link to it?

Reply:
sure, thanks!